src/math/p5.Vector.js

```diff
@@ -3901,6 +3901,22 @@ p5.Vector = class {
  * @method clampToZero
  * @return {p5.Vector} with components very close to zero replaced with zero.
  * @chainable
+ * @example
+ * <div>
+ * <code>
+ * function setup() {
+ *   // Create a 2D vector where the x-component is near-zero
+ *   let v = createVector(0.00000000000000005, 5 );
+ *
+ *   console.log('Before:', v.x , v.y);
+ *
+ *   // Clamp negligible value of x-component to zero
+ *   v.clampToZero();
+ *   console.log('After:', v.x , v.y);
```

Comment:
If you run `grunt yui:dev`, the user won't be able to see the log values on the reference page. I think they have to open their browser developer tools to see that.